app/views/layouts/_perf_options.html.haml

```diff
@@ -27,6 +27,7 @@
                         "data-miq_sparkle_on" => true,
                         :class    => "selectpicker")
           :javascript
+             miqInitSelectPicker();
```

Comment:
@simon3z i think we don't need call to miqInitSelectPicker() for each select, all we have to do is call it once and that should initialize all selects. lets move https://github.com/simon3z/manageiq/blob/cc1059285a1160689edec22665d371906c4ad0ef/app/views/layouts/_perf_options.html.haml#L22 out to be somewhere common instead of just inside that elsif block which is why all selects work fine on VM C&U screen. we can try to add following maybe at https://github.com/simon3z/manageiq/blob/cc1059285a1160689edec22665d371906c4ad0ef/app/views/layouts/_perf_options.html.haml#L32
```
:javascript
   miqInitSelectPicker();
```